typescript/infra/scripts/warp-routes/monitor/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { rootLogger } from '@hyperlane-xyz/utils';
+
+export const logger = rootLogger.child({ module: 'warp-balance-monitor' });
+
+export async function gracefullyHandleError(
```

Comment:
This seems like an odd pattern to me. IMO `gracefullyHandleError(...)` isn't more clear than a try/catch around the call. personally, I would opt for making the calls themselves swallow the error if that's the desired behavior. But if you do want a general 'catcher' util, it could be moved up and called just `try()`

Reply:
yeah agree it's a little awkward - the main motivation was wanting to do some things concurrently in `updateTokenMetrics` and not wanting a lot of nested try/catches. I'll rename to `tryFn` for now